art/art_dic.py

```diff
@@ -272,7 +279,9 @@
     "facepalm": "(>ლ)",
     "yo": "__o000o__(o)(o)__o000o__",
     "badass": "(⌐■_■)--︻╦╤─ - - -",
-    "cigaret": "()___)____________)",
+    "cigarette1": "()___)____________)",
+    "cigarette2": "(____((____________()~~~",
+    "cigarette3": "(̅_̅_̅_̅(̅_̅_̅_̅_̅_̅_̅_̅_̅̅_̅()ڪے",
```

Comment:
could you please change `cigarette1` and `cigarette3`?
`cigarette3` seems to be a more realistic model for cigarette :smoking:

Reply:
Good point!
Done https://github.com/sepandhaghighi/art/pull/92/commits/750a601862b9da3bb14114027c5d0e6ae8f5e3a9